cli/azd/resources/scaffold/base/modules/set-redis-conn.bicep

```diff
@@ -0,0 +1,41 @@
+param name string
+param keyVaultName string
+param passwordSecretName string
+param urlSecretName string
+
+resource redisConn 'Microsoft.Cache/redis@2024-03-01' existing = {
+  name: name
+}
+
+resource keyVault 'Microsoft.KeyVault/vaults@2022-07-01' existing = {
+  name: keyVaultName
+}
+
+resource passwordSecret 'Microsoft.KeyVault/vaults/secrets@2022-07-01' = {
+  name: passwordSecretName
+  parent: keyVault
+  properties: {
+    contentType: 'string'
+    attributes: {
+      enabled: true
+      exp: 0
+      nbf: 0
```

Comment:
is this mandatory?  
And, if yes, should we have parameters for this with default values?

Reply:
removed these properties since they correspond with the default values.